.github/workflows/static_checks.yml

```diff
@@ -114,3 +114,18 @@ jobs:
         ! -name update-go-deps-shas \
         ! -name web \
         | xargs -I {} bin/shellcheck -x {}
+  markdown_lint:
+    name: Markdown lint
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Checkout code
+      # actions/checkout@v2
+      uses: actions/checkout@722adc6
+    - name: Markdown lint
+      # nosborn/github-action-markdown-cli@v1.1.1
+      uses: nosborn/github-action-markdown-cli@v1.1.1
```

Comment:
rather than using this off-the-shelf action, should we just use `bin/markdownlint`?  that way we wouldn't need to worry about keeping these in sync.

Reply:
Yep good idea. I changed the action to use `bin/markdownlint`

Reply:
Example of it running: https://github.com/kleimkuhler/linkerd2/pull/21/checks?check_run_id=679879798

Reply:
I don't know if npm guarantees that if one vets some tool's code at some version, that code for that same version won't be changed down the line with something bad. I like using github actions pointing to a sha (and not a version tag) whenever possible because you get that guarantee. This is just a tioli comment though :-)

Reply:
a SHA is more in my comfort zone too